### PR TITLE
Fallback to pre-built AMI images if none privately built

### LINKED ISSUE #415: Provide prebuilt AMI image per region
**What happened**:

We currently rely on the user to build their packer images. This takes a while and will potentially a bad first impression with new users

**What you expected to happen**:

I would like to have some AMI published with every release that are used by default, if no local AMI is found.

### PR DESCRIPTION
**Pull request by JoshVanL**
**What this PR does / why we need it**:
If EBS is encrypted and no images have been privately built then fallback to using Jetstack's publicly built image for that version. 

fixes #415 

```release-note
Fallback to using Jetstack's pre-built public AMI if none are privately built.
```

/assign @simonswine

### CODE REVIEW

File: pkg/packer/packer.go [hunk 1]
```diff
@@ -116,5 +117,21 @@ func (p *Packer) IDs(encrypted bool) (map[string]string, error) {
...
+
+		image, err := p.tarmak.Cluster().Environment().Provider().DefaultImage(version)
+		if err != nil {
+			return nil, err
+		}
+
+		imagesChangeTime[image.BaseImage] = image.CreationTimestamp.Time
+		imageIDByName[image.BaseImage] = image.Name
+
+		p.log.Warnf("EBS is unencrypted so using Jestack's pre-built default image from version %s",
```
Comment (simonswine) [inline pkg/packer/packer.go]: Spelling of jetstack

Comment (JoshVanL) [reply pkg/packer/packer.go]: Good catch, not the best thing to misspell...

**Pull request merged**